src/app/centroidController.js

```diff
@@ -1,39 +1,43 @@
 define([
-    'app/config',
```

Comment:
These should be spaces, not tabs. Tried to figure out how to enforce this via jscs but was unsuccessful. Is AMD Butler inserting tabs for you? It should be using spaces.